parser/parser.go

```diff
@@ -162,6 +160,87 @@ func (p *parser) parseCommaList(end tokenKind, elementKind string) (*token, ast.
 	}
 }
 
+func (p *parser) parseArgument() (*ast.Identifier, ast.Node, error) {
+	var id *ast.Identifier
+	if p.peek().kind == tokenIdentifier && p.doublePeek().kind == tokenOperator && p.doublePeek().data == "=" {
+		ident := p.pop()
+		var tmpID = ast.Identifier(ident.data)
+		id = &tmpID
+		p.pop() // "=" token
+	}
+	expr, err := p.parse(maxPrecedence)
+	if err != nil {
+		return nil, nil, err
+	}
+	return id, expr, nil
+}
+
+// TODO(sbarzowski) - this returned bool is weird
+// TODO(sbarzowski) - name - it's also used for parameters
+func (p *parser) parseArguments(end tokenKind, elementKind string) (*token, *ast.Arguments, bool, error) {
```

Comment:
Name could be "parseCommaList"

Reply:
Hmm... it was like that before, but it's not just a comma-separated list of expressions. It parses a special format of arguments and parameters.